src/components/add-wine/AddWineForm.js

```diff
@@ -13,21 +13,91 @@ import "./styles.scss";
 import * as dispatchers from "../../dispatchers";
 import ImageCheckbox from "./image-checkbox/image-checkbox";
 import { withFirebase } from "../../firebase/index";
+import useForm from "./useForm"
 // TODO: VALIDATOR
 
+
 const AddWineForm = props => {
-  const [wineName, setWineName] = useState("");
-  const [wineType, setWineType] = useState("RED");
-  const [wineYear, setWineYear] = useState("2002");
-  const [wineRegion, setWineRegion] = useState("Bordeaux");
-  const [wineCountry, setWineCountry] = useState("Frankrike");
-  const [wineGrape, setWineGrape] = useState("Pinot Noir");
-  const [sanderRating, setSanderRating] = useState(6.0);
-  const [ineRating, setIneRating] = useState(5.0);
+  // const [wineName, setWineName] = useState("");
+  // const [wineType, setWineType] = useState("RED");
+  // const [wineYear, setWineYear] = useState("2002");
+  // const [wineRegion, setWineRegion] = useState("Bordeaux");
+  // const [wineCountry, setWineCountry] = useState("Frankrike");
+  // const [wineGrape, setWineGrape] = useState("Pinot Noir");
+  // const [sanderRating, setSanderRating] = useState(6.0);
+  // const [ineRating, setIneRating] = useState(5.0);
   const [fitsTo, setFitsTo] = useState([]);
-  const [error, setError] = useState(null);
+ 
 
   // TODO: CONSIDER SET INSTEAD WITH PUSH AND POP.
+  const stateSchema = {
+    wineName: {value: '', error: ''},
+    wineType: {value: 'Red', error: ''},
+    wineYear: {value: '2002', error: ''},
+    wineCountry: {value: 'Bordeaux', error: ''},
+    wineGrape: {value: 'Frankrike', error: ''},
+    wineRegion: {value: 'Pinot Noir', error: ''},
+    sanderRating: {value: '6.0', error: ''},
+    ineRating: {value: '5.0', error: ''},
+  }
+
+  const validationSchema = {
```

Comment:
Can this constant be extracted into a separate file?

Reply:
Done